app/Services/Blink.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rogue\Services;
+
+use DoSomething\Gateway\Common\RestApiClient;
+
+class Blink extends RestApiClient
+{
+    /**
+     * Create a new Blink API client.
+     */
+    public function __construct()
+    {
+        $url = config('services.blink.url') . '/v1/';
+
+        parent::__construct($url);
+    }
+
+    /**
+     * Send a POST request to be sent to Quasar.
+     *
+     * @param  array $params
+     * @return array|null
+     */
+    public function sendToQuasar($request)
+    {
+        return $this->post('events/quasar-relay', $request);
```

Comment:
Rather than creating a new `Blink` class, can we just call this directly in `SendPostToQuasar` and co.?

```php
    /**
     * Execute the job.
     *
     * @return void
     */
    public function handle(Blink $blink)
    {
        // Format the payload & send to Quasar.
        $payload = $this->post->toQuasarPayload();
        gateway('blink')->post('events/quasar-relay', $payload);
```

We can then always refactor this later to add the real method in Gateway.

Reply:
Oh sure! @sbsmith86 what do you think about this instead of extending `Blink.php`? I'm into this because it's one line and we'll have less stuff to change should this method end up in Gateway.

Reply:
sounds good

Reply:
I forgot that we could even do this but `Blink.php` sure does extend `RestApiClient`